sdk/src/test/java/com/kin/ecosystem/balance/presenter/BalancePresenterTest.java

```diff
@@ -85,14 +89,15 @@ public void setUp() throws Exception {
 
 		balancePresenter.onAttach(balanceView);
 		verify(balanceView).setWelcomeSubtitle();
-		verify(blockchainSource).addBalanceObserverAndStartListen(balanceObserverCaptor.capture());
+		verify(blockchainSource).addBalanceObserver(balanceObserverCaptor.capture(), sseCaptor.capture());
 		verify(orderRepository).addOrderObserver(orderObserverCaptor.capture());
+		assertTrue(sseCaptor.getValue());
```

Comment:
Couuld be good to add to other tests  assertFalse(sseCaptor.getValue()); - making sure it no one will change it